backend/models/website_app_config.py

```
@@ -0,0 +1,306 @@
+        
+        # Define allowed file extensions
+        allowed_extensions = ('.ico', '.png', '.svg')
+        
+        # Check if it's an asset URL (bypass extension validation for these)
+        if v.startswith('/api/v1/assets/'):
+            # Validate basic asset URL format
+            asset_url_pattern = r'^/api/v1/assets/(public/id/[a-f0-9]+|[a-zA-Z0-9_/\-]+\.(ico|png|svg))$'
+            if re.match(asset_url_pattern, v):
+                return v
+            else:
+                raise ValueError('Invalid asset URL format')
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)